hle-survey/transformation/src/test/scala/org/broadinstitute/monster/dap/DogTransformationsSpec.scala

```diff
@@ -0,0 +1,29 @@
+package org.broadinstitute.monster.dap
+
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+import java.time.LocalDate
+
+class DogTransformationsSpec extends AnyFlatSpec with Matchers {
+  behavior of "DogTransformations"
+
+  // TODO add tests for dog demographics transformation
+
+  it should "correctly map study status fields" in {
+    val exampleDogFields = Map[String, Array[String]](
+      "dd_us_born" -> Array("1"), // a required field
```

Comment:
Why is this required here?

Reply:
It looks like one of the transformation methods uses getRequired for that field without being wrapped in a conditional. I had assumed it was intentional but maybe not? (the getRequired call is [here](https://github.com/DataBiosphere/dog-aging-ingest/blob/efc2a9f8f7acb2a3b34901412f1b23911478b831/hle-survey/transformation/src/main/scala/org/broadinstitute/monster/dap/DogTransformations.scala#L257)).

Reply:
Ah! Would change this test to call `mapStudyStatus` instead, so we can remove this.